libafl/src/observers/map.rs

```diff
@@ -70,8 +70,7 @@ fn init_count_class_16() {
 }
 
 /// Compute the hash of a slice
-fn hash_slice<T>(slice: &[T]) -> u64 {
-    let mut hasher = RandomState::with_seeds(0, 0, 0, 0).build_hasher();
+fn hash_slice<T, H: Hasher>(slice: &[T], hasher: &mut H) -> u64 {
```

Comment:
This entire function may replaced with `.as_slice().hash(&mut hasher)`, and should be deleted.

Reply:
`hash_slice` should be renamed to `hash_helper`. Its used to define the `hash`  function. I can't see how to remove it.

Reply:
@addisoncrump any last words on this? Otherwise we'll merge

Reply:
It can be removed. One sec.